src/App.tsx

```diff
@@ -1,14 +1,36 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodoById } from './utils/getTodoById';
+import { TodoCompletedCategory } from './types/todoCompletedCategory';
+import { setTodosFromApi } from './utils/setTodosFromApi';
+import { filterTodosByQuery } from './utils/filterTodosByQuery';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [visibleTodos, setVisibleTodos] = useState<Todo[]>([]);
+  const [electTodoId, setElectTodoId] = useState<number>(0);
+  const [loading, setLoading] = useState<boolean>(true);
```

Comment:
- Boolean variables should start from `is` or `has`
- You should show loader only when you actually loading something